src/runtime/hexagon_remote/dlib.cpp

```diff
@@ -495,25 +496,25 @@ void *mmap_dlopen(const void *code, size_t size) {
 void *mmap_dlsym(void *from, const char *name) {
     if (!from) return NULL;
```

Comment:
Isn't RTLD_DEFAULT == NULL, at least on some platforms? https://code.woboq.org/userspace/glibc/dlfcn/dlfcn.h.html#_M/RTLD_DEFAULT

I think this check should go after the RTLD_SELF/RTLD_DEFAULT check.

Reply:
Interesting...on Hexagon it is -2 (HEXAGON_Tools/8.2.07/Tools/target/hexagon/include/dlfcn.h).  Seems like a good change to make though just to be safe.